packages/flutter_web_plugins/pubspec.yaml

```diff
@@ -4,7 +4,7 @@ author: Flutter Authors <flutter-dev@googlegroups.com>
 homepage: http://flutter.dev
 
 environment:
-  sdk: ">=2.0.0-dev.28.0 <3.0.0"
+  sdk: ">=2.10.0-0.0.dev <3.0.0"
```

Comment:
Isn't this constraint only enough to opt-in if its on an allow-list? I didn't think it was...